server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -21,6 +23,17 @@ def validate(deployment)
         end
       end
 
+      def validate_only_one_deployment(deployment)
+        other_running_deployments = ::Fusor::Deployment.joins(:foreman_task)
+                                      .where.not(id: deployment.id)
+                                      .where(foreman_tasks_tasks: {state: 'running'})
+
+        unless other_running_deployments.empty?
+          other = other_running_deployments.first
+          deployment.errors[:foreman_task_uuid] << _("Deployment #{other.id}: #{other.name} is already running")
+        end
+      end
+
```

Comment:
ruby looks good. My biggest worry is we have to wait for one deployment to completely finish before being able to kick the other one off. The biggest issue I see with multiples running at the same time is manage_content failing.  But if we truly only support one at a time, this is it.